src/libutil/package.nix

```diff
@@ -65,6 +82,7 @@ mkMesonLibrary (finalAttrs: {
   mesonFlags = [
     (lib.mesonEnable "cpuid" stdenv.hostPlatform.isx86_64)
   ];
+  NIX_CFLAGS_COMPILE = "-I${widecharwidth}/include";
```

Comment:
Than we also shouldn't need this.

Reply:
Probably needs a `pkg-config` file then.

Ideally we could add that upstream.
Also this should be packaged in Nixpkgs, but developing the expression here is quite convenient.

Reply:
I updated the code. Would the current definition of the derivation be ready for nixpkgs?